ext/scripts/install_scripts/install_via_pip.pl

```diff
@@ -15,6 +15,8 @@ =head1 SYNOPSIS
     --with-versions                       Uses versions specified in the input file in the second element of each line
     --allow-no-version                    If --with-versions is active, allow packages to have no version specified
     --allow-no-version-for-urls           If --with-versions is active, allow packages specified by urls to have no version
+    --ignore-installed                    Set the --ignore-installed option for pip
+    --use-deprecated-legacy-resolver      Set the --use-deprecated=legacy-resolver option for pip
```

Comment:
This sounds a little hacky. Maybe we should create a ticket so that we don't forget to review if this is necessary in future releases.